packages/console/src/components/ToggleThemeButton/index.module.scss

```diff
@@ -0,0 +1,32 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  .button {
+    padding: _.unit(2);
+  }
+
+  .icon {
+    color: var(--color-text-secondary);
+  }
+
+  &.large {
+    .icon {
+      width: 24px;
+      height: 24px;
+    }
+  }
+
+  &.medium {
+    .icon {
+      width: 20px;
+      height: 20px;
+    }
+  }
+
+  &.small {
+    .icon {
+      width: 16px;
+      height: 16px;
+    }
+  }
```

Comment:
Is it being used anywhere else?

Reply:
It will be used on the onboarding sie page. (WIP)